### PR TITLE
fix test of affine_grid with rocm

### PR DESCRIPTION
**Pull request by Ray2020BD**
### PR types

Bug fixes
### PR changes

APIs
### Describe

fix test of affine_grid with rocm

image

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_affine_grid_op.py [hunk 1]
```diff
@@ -83,6 +83,8 @@ def initTestCase(self):
         self.output_shape = np.array([20, 2, 5, 7]).astype("int32")
         self.dynamic_shape = True
         self.use_cudnn = True
+        if paddle.fluid.core.is_compiled_with_rocm():
+            self.use_cudnn = False  # Current cudnn is not supported in this op with rocm
```
Comment (qili93) [inline python/paddle/fluid/tests/unittests/test_affine_grid_op.py]: 注释改成 # ROCM platform do not have MIOPEN kernel for affine_grid

**Comment by qili93** [comment]
affine_grid API的use_cudnn也需要根据判断是否是rocm平台来关掉use_cudnn

File: python/paddle/fluid/tests/unittests/test_affine_grid_op.py (hunk 1 above)
Comment (Ray2020BD) [reply python/paddle/fluid/tests/unittests/test_affine_grid_op.py]: done

**Comment by qili93** [comment]
@Ray2020BD 贴一下rocm环境下这两个单测的结果
test_affine_grid_op
test_affine_grid_function

**Review by wanghaoshuang — APPROVED** [review]

**Comment by Ray2020BD** [comment]
> @Ray2020BD 贴一下rocm环境下这两个单测的结果
> test_affine_grid_op
> test_affine_grid_function

done.

**Review by qili93 — APPROVED** [review]
LGTM

**Review by zhiqiu — APPROVED** [review]
LGTM

**Pull request merged**